docs/docs/Data Validation/Known Issues.md

```diff
@@ -9,154 +9,187 @@ We maintain a [list of these known issues on GitHub](https://github.com/singular
-65498                  -1.000000            -1.000000                            -1.000000
-2023-12-23 11:27:47,744 [WARNING] oge.oge.validation:82 EIA-923 Input Totals:
-2023-12-23 11:27:47,744 [WARNING] oge.oge.validation:83 
-              net_generation_mwh  fuel_consumed_mmbtu  fuel_consumed_for_electricity_mmbtu
-plant_id_eia                                                                              
-1316                      505.00              17176.0                              17176.0
-10613                  373515.00            9084933.0                            1544644.0
-50937                       0.00               1103.0                               1103.0
-54724                   98804.00             337120.0                             337120.0
-54809                    9033.11            1493143.0                             567067.0
-58256                     583.00               2274.0                               2274.0
-59817                    1798.00               6138.0                               6138.0
-59825                    9972.00              34024.0                              34024.0
-65498                    1158.00               3950.0                               3950.0
-2023-12-23 11:27:47,744 [WARNING] oge.oge.validation:86 Allocated Totals:
-2023-12-23 11:27:47,744 [WARNING] oge.oge.validation:87 
-              net_generation_mwh  fuel_consumed_mmbtu  fuel_consumed_for_electricity_mmbtu
-plant_id_eia                                                                              
-1316                      541.00              17176.0                              17176.0
-10613                  373515.00            9136149.0                            1595860.0
-50937                       0.00                969.0                                969.0
-54724                       0.00                  0.0                                  0.0
-54809                    9033.11            1040724.0                             114648.0
-58256                     666.00               2274.0                               2274.0
-59817                    1700.00               5802.0                               5802.0
-59825                       0.00                  0.0                                  0.0
-65498                       0.00                  0.0                                  0.0
+For 7 plants, the total generation or fuel allocated to individual generators does not match the total generation or fuel associated with the plant in the raw input data. Possible causes include inconsistent reporting of prime movers or fuel types between EIA-923 tables, mid-year plant retirements, anomalous fuel totals being reported, or other unknown issues. 
+
+```log
+2024-04-03 12:47:46,436 [WARNING] oge.oge.validation:128 There are 7 plant/fuel combinations with non-zero (missing) net generation or fuel consumed that are different in aloocated EIA-923 and EIA-923 Input
```

Comment:
"aloocated" is mis-spelled here. We should update this in the logger text.

Reply:
Done